__tests__/app.test.js

```diff
@@ -130,6 +139,75 @@ describe("GET /api/articles/:article_id", () => {
       .expect(400)
       .then(({ body }) => {
         const { msg } = body;
+
+        expect(msg).toBe("Bad Request - Invalid ID");
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
```

Comment:
Really nice test as you are checking length and also making sure they all have an article_id as provided